### ISSUE TITLE
Minor changes for Packaging APIs MVP

### ISSUE THREAD

**Issue by vibhas** [carvel-dev/kapp-controller#237]
**Describe the problem/challenge you have**
We identified several small changes that need to be done for the packaging APIs MVP.

- [x]  Bump imgpkg to the version with performance enhancements
- [x]  Update examples with correct packaging concepts
- [x]  Run kbld on package repository bundle used in e2e tests (currently the imageslock is empty)
- [x]  Refine delete finalizer names for pkgrepo and pkginstall
- [x]  PackageRepository CRD should have annotation with the namespace for global packages
- [x]  Update PackageInstall's packageVersionRef => packageRef
- [x]  Change Package's spec.releasedAt to metav1.Time

Some things that still need more consideration:
- [x]  Should we add --wait-timeout=30s for pkg repo deploys? (seems like we don't ever wait it to get stuck for 15min) + --kube-api-qps/--kube-api-burst
- [ ]  Should we add waiting behavior to kapp for package repos/installs?

**Comment by ewrenn8** [comment]
waiting on new imgpkg release

**Comment by aaronshurley** [comment]
> Should we add waiting behavior to kapp for package repos/installs?
Extracting this to its own issue in kapp: https://github.com/vmware-tanzu/carvel-kapp/issues/237

**Comment by ewrenn8** [comment]
Included in `v0.20.0`
